komodo-parent/pom.xml

```diff
@@ -1,11 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<parent>
-		<groupId>org.jboss.integration-platform</groupId>
-		<artifactId>jboss-integration-platform-parent</artifactId>
-		<version>6.0.1.Final</version>
-		<relativePath></relativePath>
-	</parent>
+      <groupId>org.jboss</groupId>
+      <artifactId>jboss-parent</artifactId>
+      <version>19</version>
+      <relativePath></relativePath>
+    </parent>
```

Comment:
I realise teiid was under jboss-parent but komodo was asked to be put under integration-platform-parent by QE. Is there a reason this needs to be under one or the other?

Reply:
To align with later Teiid/WildFly community.  With the assumption that we are moving towards swarm.  Also the build was failing with an httpclient related error in the s3 resource-adapter.  This appeared to be the right direction to resolve that.  Another approach would be to look at using a 7.0.x version of the integration-platform-parent, but since that seems to diverge from WildFly this seemed better.

Reply:
Cool. This kind of design decision needs to be logged in the commit message, please.